setup.py

```diff
@@ -6,7 +6,9 @@
     name='api-validator-python',
     version=elife_api_validator.__version__,
     description='Utility for accessing and validating eLife API JSON Schemas.',
-    packages=['elife_api_validator'],
+    packages=['elife_api_validator',
+              'elife_api_validator.schemas',
+              'elife_api_validator.validators'],
     package_data={
         'elife_api_validator/schemas': '*.json'
```

Comment:
This appears not to work too, reading the docs the key should be the package name (`elife_api_validator.schemas`)?

Reply:
https://github.com/elifesciences/profiles/pull/89 breaks on both. Changing to this PR sees the `packages` work, then it doesn't find the schemas.

Reply:
I have switched to decaring non python files in a `MANIFEST.ini` like I do in `proofreader-python`.

Is now including the `json` files when I build it locally.

Reply:
👍 Tests now green here.